lib/http/form_data/multipart.rb

```diff
@@ -24,7 +26,7 @@ def initialize(data)
       #
       # @return [String]
       def content_type
-        "multipart/form-data; boundary=#{@boundary}"
+        "multipart/form-data; boundary=#{boundary}"
```

Comment:
Although we expose boundary reader, there's no real need in using it internally.
Thus, please leave this as instance variable reference.